### PR TITLE
Added OS agnostic interrupts.

### PR DESCRIPTION
**Pull request by brandonforster**
Issue #995

Signed-off-by: Brandon Forster <brandonforster@gmail.com>

### CODE REVIEW

File: internal/core/command/init.go [hunk 1]
```diff
@@ -189,7 +189,7 @@ func listenForConfigChanges() {
 	registryClient.WatchForChanges(registryUpdates, registryErrors, &WritableInfo{}, internal.WritableKey)
 
 	signals := make(chan os.Signal)
-	signal.Notify(signals, syscall.SIGINT, syscall.SIGTERM)
+	signal.Notify(signals, syscall.SIGINT, syscall.SIGTERM, os.Interrupt)
```
Comment (tsconn23) [inline internal/core/command/init.go]: In this case and others like it, just replace `syscall.SIGINT` with `os.Interrupt`. If you look at the definitions in the std lib, the latter points to the former.

**Comment by tsconn23** [comment]
recheck

**Comment by tsconn23** [comment]
recheck

**Comment by brandonforster** [comment]
recheck

**Comment by tsconn23** [comment]
recheck

**Pull request merged**